fmralign/pairwise_alignment.py

```diff
@@ -101,7 +98,12 @@ def fit_one_piece(X_i, Y_i, alignment_method):
                                        OptimalTransportAlignment,
                                        DiagonalAlignment)):
         alignment_algo = clone(alignment_method)
-    alignment_algo.fit(X_i, Y_i)
+    try:
+        alignment_algo.fit(X_i, Y_i)
+    except UnboundLocalError:
+        warn = "Unrecognized alignment method {}.".format(alignment_method) + \
+               " Please provide a recognized alignment method."
+        raise NotImplementedError(warn)
```

Comment:
Maybe this would deserve a raise error test?

Reply:
+1